pkg/metrics/collector_test.go

```diff
@@ -472,16 +484,16 @@ var _ = Describe("ResourcesMetricsCollector", func() {
 			const expected = `
         # HELP trivy_resource_configaudits Number of failing resource configuration auditing checks
         # TYPE trivy_resource_configaudits gauge
-        trivy_resource_configaudits{name="configmap-test",namespace="some-ns",resource_kind="ConfigMap",resource_name="test",severity="Critical"} 0
```

Comment:
`summary report` test should remain , info test should be separate test

Reply:
@chen-keinan I'm not to clear on what changes to make here

Reply:
I saw that override perv. summary tests  with new info tests.
just keep the old summary tests and add new tests for info data

Reply:
I've made the fix

Reply:
Resolved failing test